### ISSUE TITLE
What is a proper way to update messenger.json file?

### ISSUE THREAD

**Issue by rg4real** [tunoltd/emoji-mart-native#141]
In case it is outdated, what is a proper (recommended) way to update the contents of this file?

We can expose a script api for that purpose.

**Comment by pederjohnsen** [comment]
Hey @rg4real, those files are built using `yarn build:data` which runs this script: [build-data.js](https://github.com/tunoltd/emoji-mart-native/blob/master/scripts/build-data.js).

This is dependent on https://github.com/iamcal/emoji-data for the emoji data.

I usually try and keep these parts of the lib up to date when https://github.com/missive/emoji-mart is updated, but been low on time the last few months so we're a bit behind. During these Covid-19 times I might spend some time these next weeks getting the lib all up to date with `missive/emoji-mart` 👌 

As a side note: I think `messenger.json` is being replaced all together with just `facebook.json` as seen here: https://github.com/iamcal/emoji-data/pull/165#issuecomment-572795078

**Comment by rg4real** [comment]
Oh. Can you please add that script file into the npm package? I believe it is stripped out.
```
node scripts/build-data
internal/modules/cjs/loader.js:582
    throw err;
    ^

Error: Cannot find module '/Users/rogovdm/Documents/trackerion/node_modules/emoji-mart-native/scripts/build-data'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
    at Function.Module._load (internal/modules/cjs/loader.js:506:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:741:12)
    at startup (internal/bootstrap/node.js:285:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:739:3)
```

**Comment by pederjohnsen** [comment]
@rg4real you should be running the script like this: `yarn build:data` :)

**Comment by pederjohnsen** [comment]
@rg4real I just released `v0.6.0-beta` today which should bring support for Emoji 12.1
